gql_example_flutter/README.md

```diff
@@ -0,0 +1,16 @@
+# gql_example_flutter
+
+A new Flutter project.
```

Comment:
Could you update this to briefly show how to build `gql`-specific parts?

It's ok to expect that the readers know what Flutter is and how to run a Flutter app (in simulator/emulator or on the device).

Reply:
done in https://github.com/gql-dart/gql/pull/33/commits/fb3e77638090b3737222e9361f76987b3fedf946